cmd/notation/verify.go

```diff
@@ -246,12 +248,17 @@ func getVerifier(ctx context.Context) (notation.Verifier, error) {
 	}
 	fileCache, err := crl.NewFileCache(cacheRoot)
 	if err != nil {
-		return nil, err
+		// discard NewFileCache error directly as cache is not critical
+		logger.Debugf("failed to create file cache: %v", err)
 	}
-	crlFetcher.Cache = &clicrl.CacheWithLog{
-		Cache:             fileCache,
-		DiscardCacheError: crlFetcher.DiscardCacheError,
+	var cache corecrl.Cache
+	if fileCache != nil {
+		cache = &clicrl.CacheWithLog{
+			Cache:             fileCache,
+			DiscardCacheError: crlFetcher.DiscardCacheError,
+		}
 	}
+	crlFetcher.Cache = cache
```

Comment:
```suggestion
	if fileCache != nil {
		crlFetcher.Cache = &clicrl.CacheWithLog{
			Cache:             fileCache,
			DiscardCacheError: crlFetcher.DiscardCacheError,
		}
	}
```

Reply:
Since `fileCache` should be non-nil if `err == nil`, therefore you can have
```go
if err != nil {
    // ...
} else {
    crlFetcher.Cache = &clicrl.CacheWithLog{
        // ...
    }
}

Reply:
"Simplicity is the ultimate sophisticatio"

Thanks. Updated.